### ISSUE TITLE
[Bug] fullscreen with scale_factor 1 goes wrong on Windows [sf#289]

### ISSUE THREAD

**Issue by openMSX-import** [openMSX/openMSX#590]
**Reported by manuelbi on 2007-12-25 14:07 UTC**
At least on a few Windows systems it has been tested on, using fullscreen with a scale_factor of 1 goes wrong. The result is a small rectangle with part of the MSX screen, positioned slightly out of the middle of the screen (in the top left direction).
Hitting F10 to get the console shows it partly below that, but much bigger and hitting F10 again only partly removes it.
In other words: goes very wrong...

**Comment by openMSX-import** [comment]
**Updated by manuelbi on 2009-06-07 11:27 UTC**
- **assigned_to**: nobody --> mfeingol
- **summary**: fullscreen with scale_factor 1 goes wrong --> fullscreen with scale_factor 1 goes wrong on Windows

**Comment by MBilderbeek** [comment]
I can't reproduce it on Windows 8 anymore. Let's close it.
